pkg/logs/internal/tailers/file/tailer.go

```diff
@@ -214,8 +214,12 @@ func (t *Tailer) Stop() {
 // This is only used on UNIX.
 func (t *Tailer) StopAfterFileRotation() {
 	t.didFileRotate.Store(true)
+	bytesReadAtRotationTime := t.bytesRead
 	go func() {
 		time.Sleep(t.closeTimeout)
+		if newBytesRead := t.bytesRead - bytesReadAtRotationTime; newBytesRead > 0 {
+			log.Info("After waiting ", t.closeTimeout, "seconds after file rotation, an additional", newBytesRead, "bytes were read from file ", t.file.Path)
```

Comment:
Would be interesting to get the file size and log how much more there was remaining still. Did we get everything or not?

Reply:
Good suggestion, I've added that.